### ISSUE TITLE
Wrong version of pip installed

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#1372]
on compute canada/cedar:

~~~
./install_sct -p

Welcome to the SCT installer!
 Multiprocessing using MPI activated

SCT version ......... dev
Installation type ... git
Operating system .... linux
sudo mode ........... 0

SCT will be installed here: [/home/jcohen/sct]
Do you agree? yes/no: y
mkdir: cannot create directory ‘/home/jcohen/sct’: File exists

Removing sct and isct softlink from /home/jcohen/sct/bin
find: ‘/home/jcohen/sct/bin’: No such file or directory
find: ‘/home/jcohen/sct/bin’: No such file or directory

Skipping copy of source files (source and destination folders are the same)

Create launchers for Python scripts
>> mkdir -p /home/jcohen/sct/bin
>> cd /home/jcohen/sct/install
>> ./update_bin.sh
#!/bin/bash
$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/sct_launcher $(basename $0).py $@
yes isct_check_detection
yes isct_convert_binary_to_trilinear
yes isct_minc2volume-viewer
yes isct_test_ants
yes isct_warpmovie_generator
yes msct_image
yes msct_multiatlas_seg
yes sct_apply_transfo
yes sct_average_data_within_mask
no sct_change_image_type
yes sct_check_atlas_integrity
yes sct_check_dependencies
yes sct_compute_ernst_angle
yes sct_compute_hausdorff_distance
yes sct_compute_mscc
yes sct_compute_mtr
yes sct_compute_snr
yes sct_concat_transfo
yes sct_convert
yes sct_create_mask
yes sct_crop_image
no sct_denoising_onlm
yes sct_dice_coefficient
yes sct_dmri_compute_dti
yes sct_dmri_concat_bvals
yes sct_dmri_concat_bvecs
yes sct_dmri_create_noisemask
no sct_dmri_eddy_correct
yes sct_dmri_get_bvalue
yes sct_dmri_moco
yes sct_dmri_separate_b0_and_dwi
yes sct_dmri_transpose_bvecs
yes sct_download_data
yes sct_extract_metric
yes sct_flatten_sagittal
yes sct_fmri_compute_tsnr
yes sct_fmri_moco
yes sct_get_centerline
yes sct_image
no sct_invert_image
yes sct_label_utils
yes sct_label_vertebrae
yes sct_make_ground_truth
yes sct_maths
yes sct_merge_images
yes sct_pipeline
yes sct_process_segmentation
yes sct_propseg
yes sct_qc
yes sct_register_graymatter
yes sct_register_multimodal
yes sct_register_to_template
yes sct_resample
yes sct_segment_graymatter
yes sct_smooth_spinalcord
yes sct_straighten_spinalcord
yes sct_testing
yes sct_viewer
yes sct_warp_template

Installing conda...
>> rm -rf /home/jcohen/sct/python
>> mkdir /home/jcohen/sct/python
--2017-06-23 08:59:51--  https://repo.continuum.io/miniconda/Miniconda2-4.1.11-Linux-x86_64.sh
Resolving repo.continuum.io (repo.continuum.io)... 104.16.19.10, 104.16.18.10, 2400:cb00:2048:1::6810:130a, ...
Connecting to repo.continuum.io (repo.continuum.io)|104.16.19.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 27792209 (27M) [application/x-sh]
Saving to: ‘/tmp/tmp.J9RVSUJB11/miniconda.sh’

/tmp/tmp.J9RVSUJB11/miniconda.sh  100%[============================================================>]  26.50M  --.-KB/s    in 0.09s   

2017-06-23 08:59:52 (301 MB/s) - ‘/tmp/tmp.J9RVSUJB11/miniconda.sh’ saved [27792209/27792209]

exit status is 0
>> bash /tmp/tmp.J9RVSUJB11/miniconda.sh -p /home/jcohen/sct/python -b -f
PREFIX=/home/jcohen/sct/python
installing: python-2.7.12-1 ...
installing: conda-env-2.5.2-py27_0 ...
installing: openssl-1.0.2h-1 ...
installing: pycosat-0.6.1-py27_1 ...
installing: pyyaml-3.11-py27_4 ...
installing: readline-6.2-2 ...
installing: requests-2.10.0-py27_0 ...
installing: ruamel_yaml-0.11.14-py27_0 ...
installing: sqlite-3.13.0-0 ...
installing: tk-8.5.18-0 ...
installing: yaml-0.1.6-0 ...
installing: zlib-1.2.8-3 ...
installing: conda-4.1.11-py27_0 ...
installing: pycrypto-2.6.1-py27_4 ...
installing: pip-8.1.2-py27_0 ...
installing: wheel-0.29.0-py27_0 ...
installing: setuptools-23.0.0-py27_0 ...
Python 2.7.12 :: Continuum Analytics, Inc.
creating default environment...
using -f (force) option
installation finished.
WARNING:
    You currently have a PYTHONPATH environment variable set. This may cause
    unexpected behavior when running the Python interpreter in Miniconda2.
    For best results, please verify that your PYTHONPATH only points to
    directories of packages that are compatible with the Python interpreter
    in Miniconda2: /home/jcohen/sct/python
Conda installation successful
>> . /home/jcohen/sct/python/bin/activate /home/jcohen/sct/python

Installing Python dependencies...
Fetching package metadata .......
Solving package specifications: ..........

Package plan for installation in environment /home/jcohen/sct/python:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-env-2.6.0            |                0          502 B
    jbig-2.1                   |                0          29 KB
    jpeg-9b                    |                0         941 KB
    libffi-3.2.1               |                1          38 KB
    libgfortran-3.0.0          |                1         281 KB
    libiconv-1.14              |                0         2.0 MB
    mkl-11.3.3                 |                0       122.1 MB
    mpich2-1.4.1p1             |                0         2.7 MB
    pixman-0.34.0              |                0         3.8 MB
    xz-5.2.2                   |                1         669 KB
    libpng-1.6.27              |                0         219 KB
    libtiff-4.0.6              |                3         1.5 MB
    libxml2-2.9.4              |                0         3.7 MB
    pcre-8.39                  |                1         656 KB
    decorator-4.0.11           |           py27_0          15 KB
    enum34-1.1.6               |           py27_0          53 KB
    freetype-2.5.5             |                2         2.5 MB
    glib-2.50.2                |                1         5.6 MB
    idna-2.5                   |           py27_0         121 KB
    ipaddress-1.0.18           |           py27_0          31 KB
    mpi4py-2.0.0               |           py27_2         1.8 MB
    numpy-1.11.2               |           py27_0         6.2 MB
    olefile-0.44               |           py27_0          52 KB
    psutil-5.2.2               |           py27_0         260 KB
    pyasn1-0.2.3               |           py27_0          75 KB
    pycparser-2.17             |           py27_0         153 KB
    pyparsing-2.1.4            |           py27_0          70 KB
    pytz-2017.2                |           py27_0         204 KB
    requests-2.12.4            |           py27_0         762 KB
    sip-4.18                   |           py27_0         264 KB
    six-1.10.0                 |           py27_0          16 KB
    xlrd-0.9.4                 |           py27_0         179 KB
    xlwt-1.0.0                 |           py27_0         152 KB
    cffi-1.10.0                |           py27_0         329 KB
    cycler-0.10.0              |           py27_0          11 KB
    fontconfig-2.12.1          |                3         429 KB
    networkx-1.11              |           py27_0         1.1 MB
    pillow-4.1.1               |           py27_0         840 KB
    pip-9.0.1                  |           py27_1         1.6 MB
    python-dateutil-2.6.0      |           py27_0         232 KB
    scipy-0.18.1               |      np111py27_0        30.9 MB
    xlutils-1.7.1              |           py27_0          65 KB
    cairo-1.14.8               |                0         609 KB
    cryptography-1.6           |           py27_0         837 KB
    pandas-0.18.1              |      np111py27_0        13.0 MB
    scikit-learn-0.17.1        |      np111py27_2         8.6 MB
    harfbuzz-0.9.39            |                2         1.2 MB
    pycairo-1.10.0             |           py27_0          81 KB
    pyopenssl-16.2.0           |           py27_0          68 KB
    conda-4.3.22               |           py27_0         493 KB
    pango-1.40.3               |                1         938 KB
    qt-4.8.7                   |                4        32.7 MB
    pyqt-4.11.4                |           py27_4         3.3 MB
    matplotlib-1.5.1           |      np111py27_0         8.4 MB
    scikit-image-0.12.3        |      np111py27_1        26.4 MB
    ------------------------------------------------------------
                                           Total:       288.9 MB

The following NEW packages will be INSTALLED:

    cairo:           1.14.8-0          
    cffi:            1.10.0-py27_0     
    cryptography:    1.6-py27_0        
    cycler:          0.10.0-py27_0     
    decorator:       4.0.11-py27_0     
    enum34:          1.1.6-py27_0      
    fontconfig:      2.12.1-3          
    freetype:        2.5.5-2           
    glib:            2.50.2-1          
    harfbuzz:        0.9.39-2          
    idna:            2.5-py27_0        
    ipaddress:       1.0.18-py27_0     
    jbig:            2.1-0             
    jpeg:            9b-0              
    libffi:          3.2.1-1           
    libgfortran:     3.0.0-1           
    libiconv:        1.14-0            
    libpng:          1.6.27-0          
    libtiff:         4.0.6-3           
    libxml2:         2.9.4-0           
    matplotlib:      1.5.1-np111py27_0 
    mkl:             11.3.3-0          
    mpi4py:          2.0.0-py27_2      
    mpich2:          1.4.1p1-0         
    networkx:        1.11-py27_0       
    numpy:           1.11.2-py27_0     
    olefile:         0.44-py27_0       
    pandas:          0.18.1-np111py27_0
    pango:           1.40.3-1          
    pcre:            8.39-1            
    pillow:          4.1.1-py27_0      
    pixman:          0.34.0-0          
    psutil:          5.2.2-py27_0      
    pyasn1:          0.2.3-py27_0      
    pycairo:         1.10.0-py27_0     
    pycparser:       2.17-py27_0       
    pyopenssl:       16.2.0-py27_0     
    pyparsing:       2.1.4-py27_0      
    pyqt:            4.11.4-py27_4     
    python-dateutil: 2.6.0-py27_0      
    pytz:            2017.2-py27_0     
    qt:              4.8.7-4           
    scikit-image:    0.12.3-np111py27_1
    scikit-learn:    0.17.1-np111py27_2
    scipy:           0.18.1-np111py27_0
    sip:             4.18-py27_0       
    six:             1.10.0-py27_0     
    xlrd:            0.9.4-py27_0      
    xlutils:         1.7.1-py27_0      
    xlwt:            1.0.0-py27_0      
    xz:              5.2.2-1           

The following packages will be UPDATED:

    conda:           4.1.11-py27_0      --> 4.3.22-py27_0
    conda-env:       2.5.2-py27_0       --> 2.6.0-0      
    pip:             8.1.2-py27_0       --> 9.0.1-py27_1 
    requests:        2.10.0-py27_0      --> 2.12.4-py27_0

Fetching packages ...
conda-env-2.6. 100% |#######################################################################################| Time: 0:00:00 193.01 kB/s
jbig-2.1-0.tar 100% |#######################################################################################| Time: 0:00:00  10.22 MB/s
jpeg-9b-0.tar. 100% |#######################################################################################| Time: 0:00:00  34.76 MB/s
libffi-3.2.1-1 100% |#######################################################################################| Time: 0:00:00   9.87 MB/s
libgfortran-3. 100% |#######################################################################################| Time: 0:00:00  26.87 MB/s
libiconv-1.14- 100% |#######################################################################################| Time: 0:00:00  49.40 MB/s
mkl-11.3.3-0.t 100% |#######################################################################################| Time: 0:00:09  13.31 MB/s
mpich2-1.4.1p1 100% |#######################################################################################| Time: 0:00:00   7.40 MB/s
pixman-0.34.0- 100% |#######################################################################################| Time: 0:00:00   3.99 MB/s
xz-5.2.2-1.tar 100% |#######################################################################################| Time: 0:00:00  45.24 MB/s
libpng-1.6.27- 100% |#######################################################################################| Time: 0:00:00  32.53 MB/s
libtiff-4.0.6- 100% |#######################################################################################| Time: 0:00:00  55.91 MB/s
libxml2-2.9.4- 100% |#######################################################################################| Time: 0:00:01   3.13 MB/s
pcre-8.39-1.ta 100% |#######################################################################################| Time: 0:00:00  40.34 MB/s
decorator-4.0. 100% |#######################################################################################| Time: 0:00:00   6.51 MB/s
enum34-1.1.6-p 100% |#######################################################################################| Time: 0:00:00  16.01 MB/s
freetype-2.5.5 100% |#######################################################################################| Time: 0:00:00  44.18 MB/s
glib-2.50.2-1. 100% |#######################################################################################| Time: 0:00:01   5.52 MB/s
idna-2.5-py27_ 100% |#######################################################################################| Time: 0:00:00  21.15 MB/s
ipaddress-1.0. 100% |#######################################################################################| Time: 0:00:00  10.05 MB/s
mpi4py-2.0.0-p 100% |#######################################################################################| Time: 0:00:00  17.06 MB/s
numpy-1.11.2-p 100% |#######################################################################################| Time: 0:00:01   4.01 MB/s
olefile-0.44-p 100% |#######################################################################################| Time: 0:00:00  11.31 MB/s
psutil-5.2.2-p 100% |#######################################################################################| Time: 0:00:00  31.26 MB/s
pyasn1-0.2.3-p 100% |#######################################################################################| Time: 0:00:00  19.91 MB/s
pycparser-2.17 100% |#######################################################################################| Time: 0:00:00  27.97 MB/s
pyparsing-2.1. 100% |#######################################################################################| Time: 0:00:00  20.35 MB/s
pytz-2017.2-py 100% |#######################################################################################| Time: 0:00:00  25.98 MB/s
requests-2.12. 100% |#######################################################################################| Time: 0:00:00  38.01 MB/s
sip-4.18-py27_ 100% |#######################################################################################| Time: 0:00:00  25.44 MB/s
six-1.10.0-py2 100% |#######################################################################################| Time: 0:00:00   5.82 MB/s
xlrd-0.9.4-py2 100% |#######################################################################################| Time: 0:00:00  10.22 MB/s
xlwt-1.0.0-py2 100% |#######################################################################################| Time: 0:00:00  10.54 MB/s
cffi-1.10.0-py 100% |#######################################################################################| Time: 0:00:00  31.57 MB/s
cycler-0.10.0- 100% |#######################################################################################| Time: 0:00:00   4.67 MB/s
fontconfig-2.1 100% |#######################################################################################| Time: 0:00:00  30.85 MB/s
networkx-1.11- 100% |#######################################################################################| Time: 0:00:00   6.88 MB/s
pillow-4.1.1-p 100% |#######################################################################################| Time: 0:00:00  38.49 MB/s
pip-9.0.1-py27 100% |#######################################################################################| Time: 0:00:00  44.64 MB/s
python-dateuti 100% |#######################################################################################| Time: 0:00:00  31.12 MB/s
scipy-0.18.1-n 100% |#######################################################################################| Time: 0:00:07   4.57 MB/s
xlutils-1.7.1- 100% |#######################################################################################| Time: 0:00:00 876.20 kB/s
cairo-1.14.8-0 100% |#######################################################################################| Time: 0:00:00  34.78 MB/s
cryptography-1 100% |#######################################################################################| Time: 0:00:00   2.10 MB/s
pandas-0.18.1- 100% |#######################################################################################| Time: 0:00:01   7.52 MB/s
scikit-learn-0 100% |#######################################################################################| Time: 0:00:01   4.94 MB/s
harfbuzz-0.9.3 100% |#######################################################################################| Time: 0:00:00  40.96 MB/s
pycairo-1.10.0 100% |#######################################################################################| Time: 0:00:00  17.05 MB/s
pyopenssl-16.2 100% |#######################################################################################| Time: 0:00:00   6.30 MB/s
conda-4.3.22-p 100% |#######################################################################################| Time: 0:00:00  33.18 MB/s
pango-1.40.3-1 100% |#######################################################################################| Time: 0:00:01 940.67 kB/s
qt-4.8.7-4.tar 100% |#######################################################################################| Time: 0:00:06   5.45 MB/s
pyqt-4.11.4-py 100% |#######################################################################################| Time: 0:00:00  55.95 MB/s
matplotlib-1.5 100% |#######################################################################################| Time: 0:00:02   4.10 MB/s
scikit-image-0 100% |#######################################################################################| Time: 0:00:06   4.07 MB/s
Extracting packages ...
[      COMPLETE      ]|##########################################################################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|##########################################################################################################| 100%
Linking packages ...
[      COMPLETE      ]|##########################################################################################################| 100%
Collecting nipy==0.4.0 (from -r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 1))
  Downloading nipy-0.4.0-cp27-cp27mu-manylinux1_x86_64.whl (4.3MB)
    100% |████████████████████████████████| 4.3MB 56kB/s 
Collecting nibabel==2.1.0 (from -r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 2))
  Downloading nibabel-2.1.0.zip (4.1MB)
    100% |████████████████████████████████| 4.1MB 87kB/s 
Collecting dipy==0.11.0 (from -r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 3))
  Downloading dipy-0.11.0-cp27-cp27mu-manylinux1_x86_64.whl (13.7MB)
    100% |████████████████████████████████| 13.7MB 29kB/s 
Collecting distribute2mpi==0.3.0 (from -r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 4))
  Downloading distribute2mpi-0.3.0-py2.py3-none-any.whl
Collecting tqdm==4.11.2 (from -r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 5))
  Downloading tqdm-4.11.2-py2.py3-none-any.whl (46kB)
    100% |████████████████████████████████| 51kB 260kB/s 
Collecting sympy>=0.7.0 (from nipy==0.4.0->-r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 1))
  Downloading sympy-1.0.tar.gz (4.3MB)
    100% |████████████████████████████████| 4.3MB 80kB/s 
Collecting psutil (from distribute2mpi==0.3.0->-r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 4))
  Downloading psutil-5.2.2.tar.gz (348kB)
    100% |████████████████████████████████| 358kB 74kB/s 
Collecting dill (from distribute2mpi==0.3.0->-r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 4))
  Downloading dill-0.2.6.zip (83kB)
    100% |████████████████████████████████| 92kB 91kB/s 
Collecting mpi4py (from distribute2mpi==0.3.0->-r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 4))
  Downloading mpi4py-2.0.0.tar.gz (1.3MB)
    100% |████████████████████████████████| 1.3MB 79kB/s 
Collecting mpmath>=0.19 (from sympy>=0.7.0->nipy==0.4.0->-r /home/jcohen/sct/install/requirements/requirementsPip.txt (line 1))
  Downloading mpmath-0.19.tar.gz (498kB)
    100% |████████████████████████████████| 501kB 51kB/s 
Building wheels for collected packages: nibabel, sympy, psutil, dill, mpi4py, mpmath
  Running setup.py bdist_wheel for nibabel ... done
  Stored in directory: /home/jcohen/.cache/pip/wheels/c4/ec/00/fc60c9723b56c777ab5979a0b7575114efe2e450186de0bf28
  Running setup.py bdist_wheel for sympy ... done
  Stored in directory: /home/jcohen/.cache/pip/wheels/05/93/22/2d0f59d842347b1f38df0d3f7a3870586df60568d2a49d94c5
  Running setup.py bdist_wheel for psutil ... done
  Stored in directory: /home/jcohen/.cache/pip/wheels/65/e8/6f/b3fa73e0b422fab906cd67e8446901b27c2769a29890151135
  Running setup.py bdist_wheel for dill ... done
  Stored in directory: /home/jcohen/.cache/pip/wheels/8a/97/20/c0b851f365d12d9fb3ccf92e47b8df5b1bea5e5d2b92b962dc
  Running setup.py bdist_wheel for mpi4py ... done
  Stored in directory: /home/jcohen/.cache/pip/wheels/e4/54/93/c216e777e447e0261d961c6480f5cb0d8284e7aa1920df8fc0
  Running setup.py bdist_wheel for mpmath ... done
  Stored in directory: /home/jcohen/.cache/pip/wheels/02/2b/99/cd867d5da48d951118a8020e86c0c12a65022702426582d4b8
Successfully built nibabel sympy psutil dill mpi4py mpmath
Installing collected packages: nibabel, mpmath, sympy, nipy, dipy, psutil, dill, mpi4py, distribute2mpi, tqdm
Successfully installed dill-0.2.6 dipy-0.11.0 distribute2mpi-0.3.0 mpi4py-2.0.0 mpmath-0.19 nibabel-2.1.0 nipy-0.4.0 psutil-5.2.2 sympy-1.0 tqdm-4.11.2
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Fetching package metadata .........
Solving package specifications: .

Package plan for installation in environment /home/jcohen/sct/python:

The following packages will be UPDATED:

    numpy: 1.11.2-py27_0 --> 1.12.1-py27_0

numpy-1.12.1-p 100% |#######################################################################################| Time: 0:00:00  56.39 MB/s
Processing /home/jcohen/sct
Requirement already satisfied (use --upgrade to upgrade): nipy==0.4.0 in /home/jcohen/sct/python/lib/python2.7/site-packages (from spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): nibabel==2.1.0 in /home/jcohen/sct/python/lib/python2.7/site-packages (from spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): dipy==0.11.0 in /home/jcohen/sct/python/lib/python2.7/site-packages (from spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): distribute2mpi==0.3.0 in /home/jcohen/sct/python/lib/python2.7/site-packages (from spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): tqdm==4.11.2 in /home/jcohen/sct/python/lib/python2.7/site-packages (from spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): sympy>=0.7.0 in /home/jcohen/sct/python/lib/python2.7/site-packages (from nipy==0.4.0->spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): dill in /home/jcohen/sct/python/lib/python2.7/site-packages (from distribute2mpi==0.3.0->spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): psutil in /home/jcohen/sct/python/lib/python2.7/site-packages (from distribute2mpi==0.3.0->spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): mpi4py in /home/jcohen/sct/python/lib/python2.7/site-packages (from distribute2mpi==0.3.0->spinalcordtoolbox===dev)
Requirement already satisfied (use --upgrade to upgrade): mpmath>=0.19 in /home/jcohen/sct/python/lib/python2.7/site-packages (from sympy>=0.7.0->nipy==0.4.0->spinalcordtoolbox===dev)
Installing collected packages: spinalcordtoolbox
  Running setup.py install for spinalcordtoolbox ... done
Successfully installed spinalcordtoolbox-dev
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Installing binaries...
>> . /home/jcohen/sct/python/bin/activate /home/jcohen/sct/python
>> python ../scripts/sct_download_data.py -d binaries_debian -o /home/jcohen/sct/bin
Check folder existence...

Downloading 20170416_sct_binaries_linux.tar.gz...
Status: 100%|#####################################################################################| 81.2M/81.2M [15:00<00:00, 90.2KB/s]

Download complete

Check if folder already exists...

Unzip data to: /home/jcohen/sct/bin/

Remove temporary file...
Done!

All requirements installed!
>> . /home/jcohen/sct/python/bin/activate /home/jcohen/sct/python

Installing data...
>> rm -rf /home/jcohen/sct/data
>> mkdir -p /home/jcohen/sct/data
>> cd /home/jcohen/sct/data
>> python ../scripts/sct_download_data.py -d PAM50

Downloading 20170101_PAM50.zip...
Status: 100%|######################################################################################| 78.8M/78.8M [03:48<00:00, 344KB/s]

Download complete

Check if folder already exists...

Unzip data to: /home/jcohen/sct/data/

Remove temporary file...
Done!

>> python ../scripts/sct_download_data.py -d gm_model

Downloading 20160922_gm_model.zip...
Status: 100%|######################################################################################| 10.3M/10.3M [00:28<00:00, 356KB/s]

Download complete

Check if folder already exists...

Unzip data to: /home/jcohen/sct/data/

Remove temporary file...
Done!

>> python ../scripts/sct_download_data.py -d optic_models

Downloading 20170413_optic_models.zip...
Status: 100%|#####################################################################################| 17.7K/17.7K [00:00<00:00, 35.3MB/s]

Download complete

Check if folder already exists...

Unzip data to: /home/jcohen/sct/data/

Remove temporary file...
Done!


Do you want to add the sct_* scripts to your PATH environment? yes/no: yes



Run sct_check_dependencies...

--
Spinal Cord Toolbox (master/e94d9c28781c93a2e69db75271322da283042b99)
Running /home/jcohen/sct/scripts/sct_check_dependencies.py

OS: linux (Linux-3.10.0-514.21.1.el7.x86_64-x86_64-with-centos-7.3.1611-Core)
CPU cores: Available: 32, Used by SCT: 32
RAM: MemTotal:       264035648 kB
total        used        free      shared  buff/cache   available
Mem:         257847        3062      233967         398       20816      250832
Swap:          4095           0        4095
SCT path: /home/jcohen/sct
Installation type: git
  commit: e94d9c28781c93a2e69db75271322da283042b99
  branch: master
Check Python path...................................[OK]
Check if data are installed.........................[OK]
Check if xlwt (1.0.0) is installed..................[OK]
Check if xlutils (1.7.1) is installed...............[OK]
Check if cryptography (1.6) is installed............[OK]
Check if scikit-learn (0.17.1) is installed.........[OK]
Check if scikit-image (0.12.3) is installed.........[OK]
Check if pyqt (4.11.4) is installed.................[OK]
Check if psutil (5.2.2) is installed................[OK]
Check if matplotlib (1.5.1) is installed............[OK]
Check if pip (9.0.1) is installed...................[WARNING]
  Detected version: 8.1.2. Required version: 9.0.1
Check if requests (2.12.4) is installed.............[OK]
Check if xlrd (0.9.4) is installed..................[OK]
Check if pandas (0.18.1) is installed...............[OK]
Check if mpi4py (2.0.0) is installed................[OK]
Check if dipy (0.11.0) is installed.................[OK]
Check if distribute2mpi (0.3.0) is installed........[OK]
Check if nibabel (2.1.0) is installed...............[OK]
Check if tqdm (4.11.2) is installed.................[OK]
Check if nipy (0.4.0) is installed..................[OK]
Check if numpy is installed.........................[OK]
Check if scipy is installed.........................[OK]
Check if spinalcordtoolbox is installed.............[OK]
Check ANTs compatibility with OS ...................[OK]
Check PropSeg compatibility with OS ................[OK]
Check if figure can be opened.......................[FAIL]
(<type 'exceptions.RuntimeError'>, RuntimeError(u'Invalid DISPLAY variable',), <traceback object at 0x7f543401b560>)

INSTALLATION DONE!

Open a new Terminal window to load environment variables, or run:
 source /home/jcohen/.bashrc
~~~

**Comment by peristeri** [comment]
After testing the installation on Cedar (calculecanada) we found that the environment variable `PYTHONPATH` is set and is known to cause confusion when using conda's virtual environment. @poquirion has sent an email to calculecanada for an explanation on why the variable is set.

We will unset the `PYTHONPATH` in the installer.

**Comment by peristeri** [comment]
Solved with #1384
